serverlessworkflow/modules/ROOT/pages/cloud/common/_proc_deploy_sw_kubectl.adoc

```diff
@@ -0,0 +1,174 @@
+[[proc-deploy-sw-application-kubectl]]
+=== Deploying your workflow application using `kubectl`
+
+You can also use `kubectl` command-line interface and plain Kubernetes objects to deploy your workflow application. +
+And, instead of creating `knative` `yaml|json` descriptors, you can leverage the Quarkus Kubernetes extension and
+Kogito Knative add-on to generate the descriptors.
+
+.Prerequisites
+* Kogito Workflow application ready to use.
+* `kubectl` command-line tool is installed.
+* (Optional) Quarkus CLI is installed +
+For more information about installing the Quarkus CLI, see link:https://quarkus.io/guides/cli-tooling#installing-the-cli[Installing the Quarkus CLI].
```

Comment:
I think we have a variable for the Quarkus Guides.

Reply:
there is, thanks for spotting it.